cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -200,6 +200,13 @@
     </description>
   </property>
 
+  <property>
+    <name>zookeeper.client.timeout.millis</name>
+    <value>40000</value>
+    <description>
+      ZooKeeper client timeout in milliseconds
```

Comment:
Would "Maximum amount of time to wait for the ZooKeeper to connect to the quorum in milliseconds" be better?